src/Table.luna

```diff
@@ -71,27 +86,135 @@ class Table:
     def schema:
         wrapper = self.ptr.schema
         SchemaVal wrapper
+
+    # Returns a single `Column` value from `Table` by integer index.
+    # 
+    # > import Dataframes.Column 
+    # > import Dataframes.Types
+    # > import Dataframes.Table 
+    # > l1 = [1,2,3,4,5]
+    # > l2 = [11,12,13,14,15]
+    # > col1 = Column.fromList "col1" Int64Type l1
+    # > col2 = Column.fromList "col2" Int64Type l2
+    # > table = Table.fromColumns [col1 , col2]
+    # > column = table.columnAt 1
+    #
+    # `index`: Integer column index.
+    #
+    # `return`: `Column` value from the `Table` for given index.
+
     def columnAt index:
         columnWrapper = self.ptr.columnAt index
         Column.fromColumnWrapper columnWrapper
+    
+    # Insert a single `Column` value to `Table` under given index.
+    # 
+    # > import Dataframes.Column 
+    # > import Dataframes.Types
+    # > import Dataframes.Table 
+    # > l1 = [1,2,3,4,5]
+    # > l2 = [11,12,13,14,15]
+    # > col1 = Column.fromList "col1" Int64Type l1
+    # > col2 = Column.fromList "col2" Int64Type l2
+    # > table = Table.fromColumns [col1 , col2]
+    # > l3 = [21,22,23,24,25]
+    # > col3 = Column.fromList "col3" Int64Type l3 
+    # > newTable = table.addColumn 1 col3
+    #
+    # `index`: Integer index for newly inserted column.
+    # `column`: `Column` value to insert.
+    #
+    # `return`: `Table` value containing previous `table` with newly inserted column.
+
     def addColumn index column:
         self.fromWrapper $ self.ptr.addColumn index column.ptr
     def concat ts:
         cols = ts . each .toList . concat
         self.fromColumns cols
     def + that: Table.concat [self, that]
+
+    # Returns a `Table` value contain column matching given name.
```

Comment:
```
Returns a `Table` value containing a single column matching the given name.
```

Reply:
@mwu I have fixed the code. About the second part: since we want to have full example with fully working code in documentation for user to copy it from searcher I don't see how we can avoid code repetition? We can make bigger example to copy each time but then we need somehow say to parser and searcher that it is at some place. The actual values result should be generated from code sample but since we don't have such tools I am adding screens with the outputs.

Reply:
@sylwiabr I don't have good answer here. I just feel that either our requirements are wrong or we sorely miss necessary tooling. Doing all this by hand is very time-consuming and will impose serious maintanance burden in future. Particularly when we plan to significantly revamp the API. Perhaps we should discuss this in more detail next monday. This issue is likely out of scope for this PR, we rather need to have some bigger-scale discussion on how to best approach documentation generation.

Reply:
With this I agree. Just don't have right solution at this moment and we need desperately any docs with examples.